shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/ui/session/SessionListView.kt

```diff
@@ -64,11 +78,16 @@ internal fun SessionListView(viewModel: BaseSessionListViewModel) {
                 )
             },
         ) {
-            Column {
+            var size by remember { mutableStateOf(IntSize(0, 0)) }
+            Column(modifier = Modifier.onSizeChanged { size = it }) {
                 val days by viewModel.observeDays.observeAsState()
                 if (days?.isEmpty() != false) {
                     EmptyView()
                 } else {
+                    var selectedTabIndex by rememberSaveable { mutableStateOf(0) }
```

Comment:
I noticed selectedTab is getting lost if you leave the page and come back. Is it possible to save it to the VM?